tests/tasks/setup/kit-cluster/default.yaml

```diff
@@ -81,25 +84,62 @@ spec:
               - name: scheduler
                 args:
                   - --kube-api-qps=$(params.scheduler-qps)
+        etcd:
+          spec:
+            nodeSelector: 
+              node.kubernetes.io/instance-type: $(params.cp-instance-type)
+            containers: 
+            - name: etcd
+              args:
+                - --quota-backend-bytes=8589934592
       EOL
+      aws sts get-caller-identity
+      #Use service account attached to pod
+      # TOKEN=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+      # kubectl config set-credentials tekton --token=$TOKEN
+      # kubectl config set-context --current --user=tekton
+      #validate 
+      kubectl get pods -A
       kubectl create -f $(workspaces.config.path)/cp.yaml
-
-      cat > "$(workspaces.config.path)/dp.yaml" <<EOL
+      nodes=$(params.node_count)
+      #TODO: Assumption is always nodecount is greater than or equal to 1000 for scalability testing purposes.
```

Comment:
Do we need to make this assumption? My impression/hope was that this task would be reusable for other pipelines, but it seems like you're locking it into this scale testing case

Reply:
are you saying we want to support lesser than 1000 nodes for any testing that we do with KIT ?

Reply:
Yeah, I don't see why KIT testing needs to be restricted to clusters of 1000+ nodes, but maybe I'm missing some context